src/components/ViewRoutingSlip/PaymentInformation.vue

```diff
@@ -28,10 +28,10 @@
                 Total Amount Received
               </v-col>
               <v-col v-if="routingSlip" class="col-6 col-sm-9 status-list" data-test="total">
-                ${{ routingSlip.total && routingSlip.total.toFixed(2) }}
+                ${{ totalAmount && totalAmount.toFixed(2) }}
```

Comment:
use common util appendCurrencySymbol for uniform spacing

Reply:
thanks. updated